Library/Formula/ksh.rb

```diff
@@ -2,8 +2,11 @@
 class Ksh < Formula
   desc "ksh93, the KornShell"
   homepage "http://www.kornshell.com"
-  url "http://www2.research.att.com/~astopen/download/tgz/ast-ksh.2012-08-01.tgz",
-    :using => :nounzip, :user => "I accept www.opensource.org/licenses/eclipse:."
+  #url "http://www2.research.att.com/~astopen/download/tgz/ast-ksh.2012-08-01.tgz",
+  #  :using => :nounzip, :user => "I accept www.opensource.org/licenses/eclipse:."
+  url "http://www.opensource.apple.com/source/ksh/ksh-23/ast-ksh.2012-08-01.tgz",
+    :using => :nounzip
+  mirror "https://www.mirrorservice.org/pub/pkgsrc/distfiles/ast-ksh.2012-08-01.tgz"
```

Comment:
Probably the former, especially if we can temporarily ditch the `:using => :nounzip` extra.